asserts/signtool/sign.go

```diff
@@ -108,3 +114,25 @@ func Sign(opts *Options, keypairMgr asserts.KeypairManager) ([]byte, error) {
 
 	return asserts.Encode(a), nil
 }
+
+type KeypairManager interface {
+	asserts.KeypairManager
+
+	GetByName(keyNname string) (asserts.PrivateKey, error)
+	Export(keyName string) ([]byte, error)
+	List() ([]asserts.ExternalKeyInfo, error)
+	DeleteByName(keyName string) error
+}
+
+func GetKeypairManager() (KeypairManager, error) {
```

Comment:
this needs a doc comment

Reply:
I'm a bit unsure why this wasn't left in keymgr.go?

Reply:
> this needs a doc comment

Ah fair point, that's because I started by moving just this single method, then ended up moving everything else. Makes sense to keep it in keymgr.go.

Reply:
Done.